tests/Command/AddUserCommandTest.php

```diff
@@ -31,7 +31,7 @@ protected function setUp()
         exec('stty 2>&1', $output, $exitcode);
         $isSttySupported = 0 === $exitcode;
 
-        if ('Windows' === \PHP_OS_FAMILY || !$isSttySupported) {
+        if ('Windows' === PHP_OS_FAMILY || !$isSttySupported) {
```

Comment:
just curious. Why this?

Reply:
It's unrelated to this pull request ... but without this change, the php-cs-fixer linter shows an error. I was lazy and did the change here instead of creating a new PR 🙈